codex/node.nim

```diff
@@ -226,7 +226,12 @@ proc retrieve*(
       proc erasureJob(): Future[void] {.async.} =
         try:
           # Spawn an erasure decoding job
-          without res =? (await self.erasure.decode(manifest)), error:
+          let
+            erasure = Erasure.new(
+              self.networkStore,
+              leoEncoderProvider,
+              leoDecoderProvider)
+          without res =? (await erasure.decode(manifest)), error:
```

Comment:
It seems like `res` is bound but never used, so what happens to the decoded manifest?

Reply:
We already have a decoded manifest, this is reconstruction happening on download, but `res` should be `_`...